.github/workflows/TestStatic.yml

```diff
@@ -5,6 +5,9 @@ on:
       - main
   pull_request:
     types: [opened, synchronize, reopened]
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Is there an option for "random 24 hour period"? Or does github already load balance these things?

Reply:
I believe that GitHub already does their own load balancing on schedules, you'll often see them occur at off-minutes around the requested schedule.